internal/tools/orchestrator/scheduler/executor/plugins/k8s/deployment.go

```diff
@@ -660,6 +679,17 @@ func (k *Kubernetes) newDeployment(service *apistructs.Service, serviceGroup *ap
 
 	SetPodAnnotationsBaseContainerEnvs(deployment.Spec.Template.Spec.Containers[0], deployment.Spec.Template.Annotations)
 
+	err = setCoreErdaLabels(serviceGroup, service, deployment.Labels)
```

Comment:
In addition to the workload label, the pod template also needs to be maintained.